src/types.ts

```diff
@@ -77,3 +77,20 @@ export enum PERMISSION_LEVELS {
   WRITE = 'write',
   ADMIN = 'admin',
 }
+
+export type PartialChatMessage = {
+  chatId: string;
+  body: string;
+};
+
+export type ChatMessage = {
+  chatId: string;
+  creator: string;
+  createdAt: string;
+  body: string;
+};
+
+export interface Chat {
+  id: string;
+  messages: Array<ChatMessage>;
+}
```

Comment:
Are we gonna have these types shared somewhere? It might be useful for `graasp-chatbox`